pom.xml

```diff
@@ -30,6 +34,18 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-webflux</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
@amerkul sort your dependencies

test dependencies should be located at the bottom of `dependencies` XML node